Core/BaseGameRules.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Immutable;
+
+namespace PCSharpGen.Core
+{
+    public abstract class BaseGameRules
+    {
+        public abstract ImmutableHashSet<string> StackedBonuses { get; }
```

Comment:
The data files actually already have a way of specifying what bonuses stack and how. Do we want to start modeling like that here?

Reply:
Yeah, we can have a Lua/LstGameRule or something that just reads it from wherever, and then we can delete this one.  For now it's useful to be able to play around with something before we have all the LUA stuff in place.  I'll add a placeholder comment or something. :-)

Reply:
I added a big comment saying it's just a placeholder.